lib/jsdom/living/aborting/AbortController-impl.js

```diff
@@ -7,8 +7,8 @@ class AbortControllerImpl {
     this.signal = AbortSignal.createImpl(globalObject, []);
   }
 
-  abort() {
-    this.signal._signalAbort();
+  abort(reason = undefined) {
```

Comment:
```suggestion
  abort(reason) {
```

since the Web IDL layer will take care of it.